dist/platforms/windows/steps/build.ps1

```diff
@@ -0,0 +1,147 @@
+
+    # Create Editor directory if it does not exist
+    if(-Not (Test-Path -Path $Env:UNITY_PROJECT_PATH\Assets\Editor))
+    {
+        # We use -Force to suppress output, doesn't overwrite anything
+        New-Item -ItemType Directory -Force -Path $Env:UNITY_PROJECT_PATH\Assets\Editor
+    }
+
+    # Copy the build script of Unity Builder action
+    Copy-Item -Path "c:\UnityBuilderAction" -Destination $Env:UNITY_PROJECT_PATH\Assets\Editor -Recurse
+
+    # Set the Build method to that of UnityBuilder Action
+    $Env:BUILD_METHOD="UnityBuilderAction.Builder.BuildProject"
+
+    # Verify recursive paths
+    Get-ChildItem -Path $UNITY_PROJECT_PATH\Assets\Editor -Recurse
+}
+
+#
+# Pre-build debug information
+#
+
+Write-Output ""
+Write-Output "###########################"
+Write-Output "#    Custom parameters    #"
+Write-Output "###########################"
+Write-Output ""
+
+Write-Output "$('"')$($Env:CUSTOM_PARAMETERS)$('"')"
+
+Write-Output ""
+Write-Output "###########################"
+Write-Output "#    Current build dir    #"
+Write-Output "###########################"
+Write-Output ""
+
+Write-Output "$('Creating "')$($Env:BUILD_PATH_FULL)$('" if it does not exist.')"
+if (-Not (Test-Path -Path $Env:BUILD_PATH_FULL)) 
+{
+    mkdir "$Env:BUILD_PATH_FULL"
+}
+Get-ChildItem $Env:BUILD_PATH_FULL
+
+Write-Output ""
+Write-Output "###########################"
+Write-Output "#    Project directory    #"
+Write-Output "###########################"
+Write-Output ""
+
+Get-ChildItem $Env:UNITY_PROJECT_PATH
+
+#
+# Build
+#
+
+Write-Output ""
+Write-Output "###########################"
+Write-Output "#    Building project     #"
+Write-Output "###########################"
+Write-Output ""
+
+& "C:\Program Files\Unity\Hub\Editor\$Env:UNITY_VERSION\Editor\Unity.exe" -quit -batchmode -nographics `
+                                                                          -projectPath $Env:UNITY_PROJECT_PATH `
+                                                                          -executeMethod $Env:BUILD_METHOD `
+                                                                          -buildTarget $Env:BUILD_TARGET `
+                                                                          -customBuildTarget $Env:BUILD_TARGET `
+                                                                          -customBuildPath $Env:CUSTOM_BUILD_PATH `
+                                                                          -buildVersion $Env:VERSION `
+                                                                          $Env:CUSTOM_PARAMETERS `
+                                                                          -logfile | Out-Host
+
+# Catch exit code
+$Env:BUILD_EXIT_CODE=$?
+
+# Display results
+if ($Env:BUILD_EXIT_CODE -eq 0) 
+{
+    Write-Output "Build Succeeded!"
+} else 
+{
+    Write-Output "$('Build failed, with exit code ')$($Env:BUILD_EXIT_CODE)$('"')"
+}
+
+# TODO: Determine if we need to set permissions on any files
+
```

Comment:
I believe this todo is now solved as everything worked?

Reply:
Everything built and ran so it looks good on my end. What did not setting permissions do on the Linux side? Did things fail to build/run?

Reply:
I see the build wasn't started yet. I've just approved the run. It always does that for first time contributors (for each repo)